js/adapt-contrib-resources.js

```diff
@@ -46,9 +46,25 @@ class Resources extends Backbone.Controller {
         resources = resources.filter(resource => resource._isGlobal !== false);
       }
       model.set('_resources', resources);
+
+      this.setupTypes(model);
+
       drawer.triggerCustomView(new ResourcesView({ model }).$el);
     });
   }
+
+  setupTypes(model) {
+    const defaultTypes = ['all', 'document', 'media', 'link']; // must contain 'all'
+
+    const customTypes = [];
+    for (let i = 1; i <= 10; i++) {
+      customTypes.push(`custom${i}`);
+    }
+
+    const types = defaultTypes.concat(customTypes);
+
+    model.set('_resourceTypes', types);
+  }
```

Comment:
@swashbuck is this the correct commit? Just wondering why this is hardcoded here? Maybe pass though resourceData and use that? Something along these lines? 

```
  setupTypes(model, resourcesData) {
    const defaultTypes = ['all']; // must contain 'all'

    const customTypes = [];
    for (let i = 1; i <= resourcesData._resourcesItems.length; i++) {
      customTypes.push(resourcesData._resourcesItems[i]._type);
    }

    const types = defaultTypes.concat(customTypes);
    
    model.set('_resourceTypes', types);
  }
```